01 Styles/02 Import Bootstrap/webpack.config.js

```diff
@@ -52,17 +52,17 @@ module.exports = {
 			}
 		]
 	},
-	plugins:[
-		new webpack.optimize.CommonsChunkPlugin('vendor', 'vendor.js'),
+	plugins: [
+		new webpack.optimize.CommonsChunkPlugin("vendor", "vendor.js"),
 		new webpack.ProvidePlugin({
-		      $: "jquery",
-		      jQuery: "jquery"
+			$: "jquery",
+			jQuery: "jquery"
 		}),
-    //Generate index.html in /dist => https://github.com/ampedandwired/html-webpack-plugin
-    new HtmlWebpackPlugin({
-      filename: 'index.html', //Name of file in ./dist/
-      template: 'index.html', //Name of template in ./src
+		// Generate index.html in /dist => https://github.com/ampedandwired/html-webpack-plugin
+		new HtmlWebpackPlugin({
+			filename: "index.html", // Name of file in ./dist/
+			template: "index.html", // Name of template in ./src
 			hash: true
-    })
```

Comment:
Are we using 2 spaces for tabs or 4? (4 can make it hard to read in a github web page)